src/domain/auth/authenticate.ts

```diff
@@ -32,8 +32,14 @@ export const loginTunnistamo = (path?: string) => {
     });
 };
 
-export const logoutTunnistamo = (path?: string) => {
-  userManager.signoutRedirect(path ? { data: { path: path } } : {});
+// TODO Decide how logout should actually work - do we really want
+// to log people out of Tunnistamo itself?
```

Comment:
Why dont we?

Reply:
When you use FB-login to log in to some random service, you don't expect to be logged out of fb when you log out of that random service. This use case is a bit similar. 

We still need to have a conversation about this with the other teams that use Tunnistamo.